Update faqs.md

jeanniefinks opened the pull request:
minor updates including the inclusion of BERT

jeanniefinks committed:
Update faqs.md

jeanniefinks committed:
Update getstarted.md

inclusion of NLP

jeanniefinks committed:
Update getstarted.md

bnellnm commented on docs/source/faqs.md:
```diff
@@ -76,7 +76,7 @@ ___
 **What is the average estimated savings for users??**
 
-This will vary but, in some cases, we are seeing 3x-10x savings. Typically, we offer 5-6x more price performance than hardware accelerators.
+We use CPUS and they are cheaper and more readily accessible than hardware accelerators. While savings will vary on the use case and business outcomes, we encourage you to benchmark our models to and measure cost savings for your specific use case.
```
This whole section seems a bit ambiguous.  We don't really define what "savings" means, i.e. time, money, power, etc.

jeanniefinks replied:
Language is provided by Marketing. I can work with @NMsasa upon his return to refine it as needed.

jeanniefinks committed:
Update faqs.md

Removing a question that could use a better response until GTM alignment is made.

tlrmchlsmth commented:
lgtm!
